app/root/root.css

```diff
@@ -1258,7 +1258,10 @@ code .comment {
 .flaky-target-chip {
   display: flex;
   margin-left: 8px;
-  gap: 8px;
+}
+
+.flaky-target-chip .icon {
+  margin-right: 8px;
```

Comment:
Thoughts on making this 0? Then it's just controlled by the gap, which is I think what's used in other contexts and looks nice.

Reply:
like this?

Reply:
yup